core/src/main/java/org/bitcoinj/script/Script.java

```diff
@@ -809,7 +799,7 @@ private static boolean castToBool(byte[] data) {
     /**
      * Cast a script chunk to a BigInteger.
      *
-     * @see #castToBigInteger(byte[], int) for values with different maximum
+     * @see #castToBigInteger(byte[], int, boolean) for values with different maximum
```

Comment:
These two changes seem to be unrelated? Maybe put them in a different commit, or even in a different PR?

Reply:
Removed